src/content/plots/StaticPlotViewPage.tsx

```diff
@@ -0,0 +1,88 @@
+import { MathJaxContext } from 'better-react-mathjax'
+import { useEffect, useState } from 'react'
+import CornerPlot from './CornerPlot'
+import downloadjs from 'downloadjs'
+import html2canvas from 'html2canvas'
+import { Button } from '@mui/material'
+import { PlotConfig, DatasetConfig, ParameterConfig } from './PlotTypes'
+import { colours } from './constants/Colours'
+
+const PlotConfigDefault: PlotConfig = {
+    plot_size: 700,
+    subplot_size: 150,
+    margin: {
+        horizontal: 10,
+        vertical: 10
+    },
+    axis: {
+        size: 100,
+        tickSize: 10,
+        ticks: 4
+    },
+    background_color: colours.plotBackground
+}
+
+function StaticPlotViewPage() {
+    {
+        /* This page will render for the route "/visualise/view/*" for any id after view/  */
+    }
+    const [config, setConfig] = useState<PlotConfig>(PlotConfigDefault)
+    const [datasetConfig, setDatasetConfig] = useState<DatasetConfig[]>([])
+    const [parameterConfig, setParameterConfig] = useState<ParameterConfig[]>([])
+
+    // Fetch plot ID assuming it is the last part of the URL
+    // E.g. Rabit/visualise/view/123
+    // Plot ID will be 123
+    const plot_id = window.location.href.split('/').slice(-1).pop()
```

Comment:
This value should be retrieved using the `useParams` hook from `react-router-dom`.